src/Umbraco.Web.UI/web.Template.config

```diff
@@ -37,6 +37,7 @@
         <add key="Umbraco.Core.TimeOutInMinutes" value="20" />
         <add key="Umbraco.Core.DefaultUILanguage" value="en-US" />
         <add key="Umbraco.Core.UseHttps" value="false" />
+        <add key="Umbraco.Core.AllowContentDashboardAccessToAllUsers" value="true"/>
```

Comment:
Is this enough to make sure it will be added to future projects, when created?

Reply:
Hmm.. 
Good question, that I don't know

Reply:
@nul800sebastiaan What is needed to ensure a new setting will be part of new projects? (but not existing projects)

Reply:
This is a setting that will only exist in new installs at the moment.

So in existing projects this will not exist. So maybe you can see if this key exists and if it does not set the default to false?

Reply:
We are already doing that in [here](https://github.com/umbraco/Umbraco-CMS/blob/56d5704167e940c540ae67864cdb5263b9b535f3/src/Umbraco.Core/Configuration/GlobalSettings.cs#L409) 🙂

Reply:
Ah perfect, so yes: it is always false unless this appsetting exists and is explicitly set to true. 

This web.template.config contains all the defaults that new sites get, so this is perfect. 

For NuGet installs: we only add anything new when it is defined in `build\NuSpecs\tools\Web.config.install.xdt` 

So in other words, this should work as expected: only brand new sites get this new key set to true.

Reply:
Great! Thanks @nul800sebastiaan 🙌